assets/i18n/fr.po

```diff
@@ -6112,60 +6134,63 @@ msgstr ""
 "arkhamdb@kamalisk.com"
 
 msgid "Requeset Account Deletion"
-msgstr ""
+msgstr "Requête de suppression de compte"
 
 msgid "Delete Account"
-msgstr ""
+msgstr "Supprimer le compte"
 
 msgid "Choose a skill"
-msgstr ""
+msgstr "Choisissez une compétence"
 
 msgid "Use the embark button below"
-msgstr ""
+msgstr "Utilisez le bouton embarquez ci-dessous"
 
 msgid "Review the world map"
-msgstr ""
+msgstr "Revoir la carte du monde"
 
 msgid "Claim previous rewards"
-msgstr ""
+msgstr "Réclamer les récompenses précédentes"
 
 msgid "Apply changes from last completed scenario or interlude"
 msgstr "Appliquer les modifications liées au dernier interlude ou scénario joué"
 
 msgid "This is a side-story location."
-msgstr ""
+msgstr "C'est une destination d'aventure annexe."
 
 #, javascript-format
 msgid "Travel cost: ${ travelDistance } time"
 msgid_plural "Travel cost: ${ travelDistance } time"
-msgstr[0] ""
-msgstr[1] ""
+msgstr[0] "Coût du voyage : ${ travelDistance } temps"
+msgstr[1] "Coût du voyage : ${ travelDistance } temps"
 
 msgid "Do you want to use the app to randomize choices?"
-msgstr ""
+msgstr "Voulez-vous utiliser l'application pour faire des choix aléatoires ?"
 
 #, javascript-format
 msgid "The cell travels from ${ departCityName } to ${ arriveCityName }."
-msgstr ""
+msgstr "La cellule se déplace de ${ departCityName } à ${ arrivéeCityName }."
```

Comment:
Be careful to not translate things inside of the `${ variable }` brackets.  I have a script to catch it, but it can lead to the app crashing.